Doc/library/cmd.rst

```diff
@@ -28,7 +28,7 @@ interface.
 
    The default, ``'tab'``, is treated specially, so that it refers to the
    :kbd:`Tab` key on every :data:`readline.backend`.
-   Specifically, if :data:`readline.backend` is ``libedit``,
+   Specifically, if :data:`readline.backend` is ``editline``,
```

Comment:
Because of 
```
Lib/site.py:447:        if readline.backend == 'editline':
Lib/test/test_pdb.py:3296:        if readline.backend == "editline":
Lib/test/test_readline.py:149:        self.assertIn(readline.backend, ("readline", "editline"))
Lib/test/test_readline.py:177:            if readline.backend == "editline":
Lib/test/test_readline.py:204:is_editline = readline.backend == "editline"
Lib/test/test_readline.py:22:    is_editline = readline.backend == "editline"
```